### PR TITLE
updating my page 

### PR DESCRIPTION
**Pull request by albeanth**
and adding in a public resume.

### CODE REVIEW

**Comment by dougwoods3** [comment]
:shipit:

File: users/albertia/index.md [hunk 1]
```diff
@@ -12,30 +12,29 @@ change here.
 
 I am a 3rd year graduate student here at OSU in Corvallis, OR working under the guidance of Todd Palmer. I began graduate school in January of 2014 and have been a full time student since. I defended my Masters of Science degree in October of 2015 and published by Masters thesis (see publications below) shortly thereafter.
```
Comment (harterj) [inline users/albertia/index.md:13]: "published **my** Master's thesis"

File: users/albertia/index.md [hunk 2]
```diff
@@ -12,30 +12,29 @@ change here.
...
 
-Since then, I have been pursuing my Doctoral degree in further developing depletion methods for MAMMOTH - the reactor physics subpackage of the Multiphysics Object Oriented Simulation Environment, otherwise known as MOOSE.  In particular, I am focusing on reducing the order of the depletion calculations in order to observe an effective speedup in depletion calculations.
+Since then, I have been pursuing my Doctoral degree in a priori parametric model reduction. In spite of progress made in mathematical methods and computer science, many problems in Nuclear Engineering suffer from what is known as the "curse of dimensionality". In standard grid based discretization techniques (such as finite elements or finite differences), the number of degrees of freedom scale exponentially.
+
+Consider the solution of a nuclear reactor dynamics problem. The solution, neutron population as a function of time, has 7 unknowns; 3 in space, 2 in direction of flight, 1 in energy, and 1 in time. Couple that with a coarse mesh of say, 10^3, and now your problem has 10^21 degrees of freedom. As one can imagine, especially for multiphysics problems this curse can get out of control, very, very quickly.
```
Comment (harterj) [inline users/albertia/index.md:17]: Isn't there a way to add in LaTeX display math for inline markdown? i.e. your 10^3 and 10^21

Comment (albeanth) [reply users/albertia/index.md:17]: @harterj I don't think there is. I did some googling on this actually and wasn't able to find anything....

File: users/albertia/index.md (hunk 1 above)
Comment (albeanth) [reply users/albertia/index.md:13]: good catch.

File: users/albertia/index.md (hunk 2 above)
Comment (dougwoods3) [reply users/albertia/index.md:17]: You could do 1e-3 for inline exponents.

Comment (albeanth) [reply users/albertia/index.md:17]: it will still look that way and not actually become an exponent, right?

Comment (dougwoods3) [reply users/albertia/index.md:17]: right. It's just an alternative to using a carrot.

**Pull request merged**